zegami_sdk/collection.py

```diff
@@ -196,12 +194,7 @@ def tags():
 
     @tags.getter
     def tags(self):
-        if self.allow_caching and self._cached_tags is not None:
-            return self._cached_tags
-        tags = self._get_tag_indices()
-        if self.allow_caching:
-            self._cached_tags = tags
-        return tags
+        return self._get_tag_indices()
```

Comment:
What was the reason for getting rid of the caching? Not to sure we should remove it entirely as this might now change behaviour in other cases.
Could we not set `allow_caching = False` for the request, or perhaps add an optional argument when getting tags to prevent the cached version?

Reply:
@ahaith  Doug commented in this PR https://github.com/zegami/zegami-python-sdk/pull/26 it might be better to have tags not cached. I'm not opposed to either way, what's your preference?

Reply:
OK, if Doug's on board then this sounds fine.